lnpbp-0024.md

```diff
@@ -59,4 +103,4 @@ Interface specification is the following Contractum code:
 
 ## Copyright
 
-This document is licensed under the Creative Commons CC0 1.0 Universal license.
+This document is licensed under the GPL-3.0 license.
```

Comment:
This left me puzzled – can you please describe motivations behind the change to GPL-3.0? Perhaps in the PR itself. Or is there some related discussion about this? Wouldn't it imply (Section 5) that much more code that builds on top of RGB24 would have to be distributed under GPL3? Thank you.

Reply:
The rationale is simple: a global naming system can't be a non-open by a definition, meaning that it's data must be available upon a request.

This standard describes an API - and not an implementation - of a such system. Thus software running it can operate on any license, while API has to be opened.